install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -454,6 +454,8 @@ data:
 {{- end }}
 {{- if and .Values.ipMasqAgent .Values.ipMasqAgent.enabled }}
   enable-ip-masq-agent: "true"
+  config: |-
```

Comment:
I think this config should be somehow specific to IP Masq agent, naming it just `config` will be confusing .

Reply:
Thanks! @joestringer 

Maybe change `config` to `ip-masq-agnet-config` something like?

Reply:
@joestringer I don't see anywhere where `enable-ip-masq-agent` is used,  Should we also put it in configMap `ip-masq-agnet`?

Reply:
```
$ git grep EnableIPMasqAgent
daemon/cmd/daemon.go:   } else if option.Config.EnableIPMasqAgent {
daemon/cmd/daemon.go:   if option.Config.EnableIPMasqAgent {
daemon/cmd/daemon_main.go:      flags.Bool(option.EnableIPMasqAgent, false, "Enable BPF ip-masq-agent")
daemon/cmd/daemon_main.go:      option.BindEnv(option.EnableIPMasqAgent)
daemon/cmd/daemon_main.go:      if option.Config.EnableIPMasqAgent {
daemon/cmd/datapath.go: if option.Config.EnableIPv4 && option.Config.EnableIPMasqAgent {
daemon/cmd/status.go:           s.IPMasqAgent = option.Config.EnableIPMasqAgent
pkg/datapath/linux/config/config.go:                    if option.Config.EnableIPMasqAgent {
pkg/datapath/maps/map.go:       if !option.Config.EnableIPMasqAgent {
pkg/option/config.go:   // EnableIPMasqAgent enables BPF ip-masq-agent
pkg/option/config.go:   EnableIPMasqAgent = "enable-ip-masq-agent"
pkg/option/config.go:   EnableIPMasqAgent          bool
pkg/option/config.go:   c.EnableIPMasqAgent = viper.GetBool(EnableIPMasqAgent)
```

Reply:
```
$ git grep EnableIPMasqAgent
daemon/cmd/daemon.go:   } else if option.Config.EnableIPMasqAgent {
daemon/cmd/daemon.go:   if option.Config.EnableIPMasqAgent {
daemon/cmd/daemon_main.go:      flags.Bool(option.EnableIPMasqAgent, false, "Enable BPF ip-masq-agent")
daemon/cmd/daemon_main.go:      option.BindEnv(option.EnableIPMasqAgent)
daemon/cmd/daemon_main.go:      if option.Config.EnableIPMasqAgent {
daemon/cmd/datapath.go: if option.Config.EnableIPv4 && option.Config.EnableIPMasqAgent {
daemon/cmd/status.go:           s.IPMasqAgent = option.Config.EnableIPMasqAgent
pkg/datapath/linux/config/config.go:                    if option.Config.EnableIPMasqAgent {
pkg/datapath/maps/map.go:       if !option.Config.EnableIPMasqAgent {
pkg/option/config.go:   // EnableIPMasqAgent enables BPF ip-masq-agent
pkg/option/config.go:   EnableIPMasqAgent = "enable-ip-masq-agent"
pkg/option/config.go:   EnableIPMasqAgent          bool
pkg/option/config.go:   c.EnableIPMasqAgent = viper.GetBool(EnableIPMasqAgent)
```